Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "hyperminhash"
-version = "0.1.4"
-authors = ["Lukas Lueg <lukas.lueg@gmail.com>"]
+version = "0.1.5"
```

Comment:
Please revert changes to the manifest, those should always happen in a separate PR